bcgov/cas-registration
# 272 alumina production

#272 alumina production page

- chore: Add in data and configurations needed for alumina production page
- feat: Add alumina production JSON Schemas and UISchemas + rebase/style/field type edits
- style: Dynamically update schema on selection of smelter tech type + UISchema and code formatting changes
- test: Add backend program config test for alumina production


Steps to verify:

- make reset_db & make run
- log in as bceid
- go to reporting/submit annual reports
- continue or start an SFO
- continue to facility information and choose Aluminum or alumina production
- continue to the next page
- should see just smelter technology type multi select and the 3 source types
- each smelter tech should add 3 annual fuel amount fields on top
- each source type when checked should add a box to add emissions
- each emission should have 1 (or 2 if anode effects) gas type
- each gas type should have multiple methodologies matching the BCIERS reporting fields excel sheet
- Anode effects - CF4 - Slope method/Overvoltage method - reporting fields should contain a field with a date widget
